packages/tx-construction/package.json

```diff
@@ -63,6 +63,8 @@
     "@cardano-sdk/input-selection": "workspace:~",
     "@cardano-sdk/key-management": "workspace:~",
     "@cardano-sdk/util": "workspace:~",
+    "@cardano-sdk/util-dev": "workspace:~",
```

Comment:
`util-dev` usually appears only in `devDependencies` (and it is already). Any reason to have it in `dependencies`?